demo/src/demos/MLP/dataUtils.ts

```diff
@@ -0,0 +1,51 @@
+import { Value, v } from "../../../../micrograd/engine";
+
+interface Data {
+  data: number[][];
+  labels: number[];
+}
+
+interface DataValues {
+  dataValues: Value[][],
+  labelValues: Value[],
+}
+
+export const generateCircleData = (
```

Comment:
We have 3 places with copies of similar code that generates generateCircleData (`circleWorker.js`, `circleWorker.ts`, `dataUtils.ts`). Can we make these data function reusable and have them in one place only (to have a single source of truth)?